### ISSUE TITLE
A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_Screenreader : The controls such as "Repository" and "License" are not announced with proper role for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-azurefunctions#2281]
**“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**

GitHub Tags:#E2E-AzureExtensionsForVSCode-Mac-Aug2020;#A11y_Azure Extensions for VS Code;#Azure Tools for VS Code;#A11yTCS;#A11yMAS;#MAC;#FTP;#Benchmark;#Screenreader;#VoiceOver;#A11ySev2:#WCAG1.3.1;#WCAG4.1.2;#External: VS Code

### Environment Details:
Application Name: Azure Extensions for VS Code
Application Version: 1.48.0-insider
Commit: cfbd1999769f4f08dce29629fb92fdc0fac53829
Date: 2020-08-06T05:40:40.871Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina Version 10.15.6 (19G73)
Screen Reader: VoiceOver

### Repro Steps:
1. Open Visual studio code
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button, the "Default Screen" will appear.
4. Start screen reader and navigate to controls such as "Repository" and "License" and observe if these are announced with role as link or not.

### Actual:
When screen reader users navigate to the controls such as "Repository" and "License" they are announced as group and text which is incorrect as on activating them web page appears on the screen so they should be announced with proper role for screen reader users.

### Expected:
The controls such as "Repository" and "License" should be announced as 'link' for screen reader users when they navigate to these controls.

### User Impact:
If proper role is not announced for the interactive elements on the screen content then the screen reader users will miss out on the information available on these elements.

### Attachment link for Reference:
[2281_A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_Screenreader The controls such as "Repository" and "License" are not announced with proper role for screen reader users.zip](https://github.com/microsoft/vscode-azurefunctions/files/5039849/2281_A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_Screenreader.The.controls.such.as.Repository.and.License.are.not.announced.with.proper.role.for.screen.reader.users.zip)

**Comment by EricJizbaMSFT** [comment]
This issue should be filed in the VS Code repo: https://github.com/Microsoft/vscode

Our extension has no control over that section of the marketplace page. For example, if you look at any extension in the marketplace, it has the exact same styling and behavior:
![Screen Shot 2020-08-10 at 11 09 01 AM](https://user-images.githubusercontent.com/11282622/89815612-ead8a580-daf9-11ea-9e13-6b6b94476989.png)

**Comment by nirmalsahoo** [comment]
Logged bug in VS Code- https://github.com/microsoft/vscode/issues/104552
As per the C & AI bug process assigning the bug to the Product Owner. Please assign this bug back to the tester once the "External" issue has been resolved and changes are pushed to the environment that will be available for bug verification.

**Comment by MayuriNaik** [comment]
@EricJizbaMSFT Logged external bug in VS Code: [108302](https://github.com/microsoft/vscode/issues/108302)

**Comment by EricJizbaMSFT** [comment]
Closing since this bug is external and due to lack of response from accessibility test team on how to get this off our tracking list.
